### PR TITLE
feat: support dbt metric filters

### PR DESCRIPTION
**Pull request by owlas**
Closes: #2203

### CODE REVIEW

**Comment by owlas** [comment]
Bumped jaffle shop to include a dbt metric with filter: https://github.com/lightdash/jaffle_shop/pull/25

**Comment by owlas** [comment]
Example usage of metrics: https://lightdash-pr-2967.onrender.com/projects/3675b69e-8324-4110-bdca-059031aa8da3/tables/orders?create_saved_chart_version=%7B%22tableName%22%3A%22orders%22%2C%22metricQuery%22%3A%7B%22dimensions%22%3A%5B%22orders_order_date_week%22%5D%2C%22metrics%22%3A%5B%22orders_completed_order_count%22%5D%2C%22filters%22%3A%7B%7D%2C%22sorts%22%3A%5B%7B%22fieldId%22%3A%22orders_order_date_week%22%2C%22descending%22%3Atrue%7D%5D%2C%22limit%22%3A500%2C%22tableCalculations%22%3A%5B%5D%2C%22additionalMetrics%22%3A%5B%5D%7D%2C%22tableConfig%22%3A%7B%22columnOrder%22%3A%5B%22orders_order_date_week%22%2C%22orders_completed_order_count%22%5D%7D%2C%22chartConfig%22%3A%7B%22type%22%3A%22cartesian%22%2C%22config%22%3A%7B%22layout%22%3A%7B%22xField%22%3A%22orders_order_date_week%22%2C%22yField%22%3A%5B%22orders_completed_order_count%22%5D%7D%2C%22eChartsConfig%22%3A%7B%22series%22%3A%5B%7B%22encode%22%3A%7B%22xRef%22%3A%7B%22field%22%3A%22orders_order_date_week%22%7D%2C%22yRef%22%3A%7B%22field%22%3A%22orders_completed_order_count%22%7D%7D%2C%22type%22%3A%22bar%22%7D%5D%7D%7D%7D%7D

File: packages/common/src/compiler/translator.ts [hunk 1]
```diff
@@ -193,6 +193,17 @@ const convertDbtMetricToLightdashMetric = (
             sql = metric.sql;
         }
     }
+    if (metric.filters && metric.filters.length > 0) {
+        const filterSql = metric.filters
+            .map(
+                (filter) =>
+                    // eslint-disable-next-line no-useless-escape
+                    `(\$\{TABLE\}.${filter.field} ${filter.operator} ${filter.value})`,
```
Comment (ZeRego) [inline packages/common/src/compiler/translator.ts:201]: Is the field always from the same table ?

**Review by ZeRego — APPROVED** [review]

File: packages/common/src/compiler/translator.ts (hunk 1 above)
Comment (owlas) [reply packages/common/src/compiler/translator.ts:201]: It's a restriction I need to document! Gonna update a bunch of the metrics docs today!

**Pull request merged**